### ISSUE TITLE
Could this component cause browser lag issues?

### ISSUE THREAD

**Issue by clandestine-avocado** [thomasloven/lovelace-layout-card#43]
Apologies if this is the wrong place - but since I added the layout-card, I've been having some serious browser performance issues. It just spins when I try to load my front end, and basically maxes out the CPU usage while trying to load the page. 

If it does load, when I am in modifying a layout-card, it seems to tax the browser then also, sometimes crashing Chrome.

**Comment by cgarwood** [comment]
I've been using layout-card for a couple weeks and just started noticing this on desktop Chrome and Android WebView. Switching to a page with layout-card works, but if you try to switch off of it the webview freezes. In the case of desktop chrome, the process for that tab will shoot up to 100% cpu and stay there until you kill the process or until Chrome kills it.

No idea what changed. It was working fine for the past week. I added a couple entities to one of my views and then it started freezing. I cleared my cache and reverted the changes I made to the lovelace files but now it freezes every time.
